uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -0,0 +1,45 @@
+define([
+  'leave-absences/shared/modules/models-instances',
+  'leave-absences/shared/models/leave-status-id-model',
+  'common/models/instances/instance'
+], function (instances) {
+  'use strict';
+
+  instances.factory('LeaveRequestInstance', [
+    'ModelInstance',
+    'LeaveRequestAPI',
+    'LeaveStatusID',
+    function (ModelInstance, LeaveRequestAPI, LeaveStatusID) {
+      return ModelInstance.extend({
+
+        /**
+         * This method is used to cancel a leave request
+         *
+         * @return {Promise}
+         */
+        cancel: function () {
+          var leaveRequest = this;
+          return LeaveStatusID.getOptionIDByName("cancelled")
```

Comment:
Please be consistent with the use of quotes, always use single quotes `''` unless otherwise necessary